about-us.html

```diff
@@ -0,0 +1,138 @@
+        </div>
+    </div>
+
+
+    <div  class="mobile-menu-contaner" id="mobile-menu-contaner">
+        <div class="close-icon">
+          <img src="images/cancel.png" alt="">
+        </div>
+          <ul class="sub-list">
+            <li ><a href="index.html">Home</a></li>
+            <li ><a href="about-us.html">About</a></li>
+            <li ><a href="#Contact">Blog</a></li>
+            <li ><a href="#Contact">Courses</a></li>
+            <li ><a href="#Contact">Sponsor</a></li>
+            <li ><a href="#Contact">News</a></li>
+
+          </ul>
+    </div>
+    <section class="about-programe">
+        <div class="content">
+            <h1>React New registration </h1>
+            <img src="images/title_underline.png" alt="underline">
+            <p class="main-content">
+                Lorem ipsum dolor sit amet consectetur, adipisicing elit. Obcaecati fugit odio ex rerum aut non iusto, ducimus modi earum quam minima nostrum accusantium sed delectus deleniti amet maxime, doloremque cum!
+            </p>
+            <p class="address">
+                Please contact us per Email for nay further questions about CC Global Summit 2015!<br>
+                Redsoftsolut@gmail.com
+            </p>
+            <p>
+                
+            </p>
+        </div>
+       
+        
+    </section>
+    <section class="entry-logo">
+       <h2> Red software Soultion Instution 2023 Logo</h2>
+       <p>
+        The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+        After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+        After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:
+       </p>
+       <img src="images/Logopit_1688405566257.png" alt="">
+       <hr>
+    </section>
+
+    <section class="past-GC">
+        <h2> See The past GC Cermoney</h2>
+        <p>
+            Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw
+        </p>
+        <div class="gc-picture">
+            <div class="left">
+                <img src="images/about_2011 1.png" alt="GC-2011">
+            </div>
+            <div class="right">
+                <img src="images/about_2013 1.png" alt="GC-2013">
+            </div>
+        </div>
+        
+     </section>
+
+     <section class="partners about">
+        <h3>Partner</h3>
+        <!-- <img class="title-image" src="images/title_underline.png" alt="underline"> -->
+        <div class="partner">
+            <img src="images/mozilla 1.png" alt="Mozilla">
+            <img src="images/googlekorea 1.png" alt="Google">
+            <img src="images/naver 1.png" alt="Naver">
+            <img src="images/daumkakaologo 1.png" alt="Daumkaka">
+            <img src="images/airbnb 1.png" alt="Airbnb">
+        </div>
```

Comment:
- Please, let's improve our design here to be close to the main design by adding the missing red line under the `Partner`
- Improve the spacing on each image to match the main design:

* our design*
![image](https://github.com/ethred/Capstone/assets/59397159/029f728b-820b-4e99-a997-45a676bf0947)


*main design*

![image](https://github.com/ethred/Capstone/assets/59397159/b35c83e1-b8d9-4bef-b281-e959b3d69167)